radix-transaction-scenarios/src/scenarios/maya_router.rs

```diff
@@ -0,0 +1,490 @@
+        ScenarioBuilder::new(core, metadata, config, start_state)
+            .successful_transaction_with_result_handler(
+                |core, config, _| {
+                    core.next_transaction_with_faucet_lock_fee(
+                        "maya-router-create-accounts",
+                        |builder| {
+                            [
+                                &config.owner_private_key,
+                                &config.swapper_private_key,
+                            ]
+                            .iter()
+                            .fold(builder, |builder, key| {
+                                builder.call_function(
+                                    ACCOUNT_PACKAGE,
+                                    ACCOUNT_BLUEPRINT,
+                                    ACCOUNT_CREATE_ADVANCED_IDENT,
+                                    AccountCreateAdvancedManifestInput {
+                                        address_reservation: None,
+                                        owner_role: OwnerRole::Fixed(rule!(require(
+                                            NonFungibleGlobalId::from_public_key(&key.public_key())
+                                        ))),
+                                    },
+                                )
+                            })
+                        },
+                        vec![],
+                    )
+                },
+                |_, _, state, result| {
+                    state
+                        .owner_account
+                        .set(result.new_component_addresses()[0]);
+                    state
+                        .swapper_account
+                        .set(result.new_component_addresses()[1]);
+                    Ok(())
+                },
+            )
+            .successful_transaction(|core, config, state| {
+                core.next_transaction_free_xrd_from_faucet(state.swapper_account.get()?)
+            })
+            .successful_transaction_with_result_handler(
+                |core, config, state| {
+                    core.next_transaction_with_faucet_lock_fee_fallible(
+                        "maya-router-create-resources",
+                        |builder| {
+                            builder.create_fungible_resource(
+                                OwnerRole::None,
+                                false,
+                                18,
+                                FungibleResourceRoles {
+                                    burn_roles: burn_roles! {
+                                    burner => rule!(allow_all);
+                                    burner_updater => rule!(deny_all);
+                                },
+                                    ..Default::default()
+                                },
+                                metadata! {
+                                    init {
+                                        "name" => "EARLY".to_owned(), locked;
```

Comment:
Can we use a different name? This will be deployed to testnets and don't want to confuse it with existing tokens.

Reply:
Done